workspaceitem-data-detect-duplicate.md

```diff
@@ -0,0 +1,232 @@
+
+The following duplicates are identified, listed inside detect-duplicate section:
+```json
+{
+  "id": 43366,
+  ...
+  "sections": {
+    ...,
+    "detect-duplicate": {
+      "matches": {
+        "9db079fc-359f-4aa4-ad4f-2dd2f0afaac6": {
+		  ...,
+          "matchObject": {
+            "id": "9db079fc-359f-4aa4-ad4f-2dd2f0afaac6",
+            "uuid": "9db079fc-359f-4aa4-ad4f-2dd2f0afaac6",
+            "name": "Sample submission",
+            "handle": "123456789/28381",
+            "metadata": [
+              {
+                "key": "dc.contributor.author",
+                "value": "Francesco, Cadili",
+                "language": null,
+                "authority": "rp05896",
+                "confidence": 600
+              },
+              {
+                "key": "dc.contributor.editor",
+                "value": "Matteo, Perelli",
+                "language": null,
+                "authority": "rp05897",
+                "confidence": 600
+              },
+              ...,
+              {
+                "key": "dc.title",
+                "value": "Sample submission",
+                "language": null,
+                "authority": null,
+                "confidence": -1
+              },
+              ...
+            ],
+            "inArchive": true,
+            "discoverable": true,
+            "withdrawn": false,
+            "lastModified": "2020-01-31T10:53:37.227+0000",
+            "type": "item"
+          }
+        }
+      }
+    },
+    ...
+  }
+}
+```
+In section detect-duplicate is listed the item (9db079fc-359f-4aa4-ad4f-2dd2f0afaac6) since the dc.title is the same as our submitting workspace item. 
+Now we can state that the found duplicate is to be further analyzed adding a comment for the administrator or ignored. 
+
+To report a note we use the following request:
+
+`curl --data 'curl --data '[ { "op": "add", "path": "/sections/detect-duplicate/matches/9db079fc-359f-4aa4-ad4f-2dd2f0afaac6/submitterDecision", "value": {"value": "verify", "note": "check the title for duplicate"}}]' -X PATCH ${dspace7-url}/api/submission/workspaceitems/43366`
+
+will result in
+```json
+{
+  "id": 43366,
+  ...
+  "sections": {
+    ...,
+    "detect-duplicate": {},
+    ...
+  }
+}
+```
+
+if instead, we wanted to ignore the duplicate we would have run the following command:
+
+`curl --data 'curl --data '[ { "op": "add", "path": "/sections/detect-duplicate/matches/9db079fc-359f-4aa4-ad4f-2dd2f0afaac6/submitterDecision", "value": {"value": "reject", "note": null}}]' -X PATCH ${dspace7-url}/api/submission/workspaceitems/43366`
+
+and we would have achieved the same result:
+```json
+{
+  "id": 43366,
+  ...
+  "sections": {
+    ...,
+    "detect-duplicate": {},
+    ...
+  }
+}
+```
+
```

Comment:
can you specify the return codes? what if a different operation than reject / verify (case sensitive) is sent?

Reply:
The return code of the two operations is updated with a description, related to the content of submitterDecision and submitterNote fields.